var/spack/repos/builtin/packages/sim-model/package.py

```diff
@@ -97,21 +97,24 @@ def _build_mods(self, mods_location, link_flag='', include_flag='',
                                mods_location)
 
         assert os.path.isfile(os.path.join(output_dir, 'special'))
-        return include_flag, link_flag
+        return include_flag_raw, link_flag_raw
+
+    def _nrnivmodlcore_params(self, inc_flags, link_flags):
+        return ['-n', self.mech_name, '-i', inc_flags, '-l', link_flags, '-V']
```

Comment:
```suggestion
        return ['-n', self.mech_name, '-i', inc_flags, '-l', link_flags]
```
I still see the issue with -V flag.